tox.ini

```diff
@@ -19,14 +19,16 @@ ignore=
 
 per-file-ignores=
     ; TYPE_CHECKING block suggestions
-    tests/*: TC001, TC002, TC003
+    tests/*: A001, A003, B010, B017, B904, C401, C405, C408, C413, C416, C420, E731, TC001, TC002, TC003, ISC001, N400, SIM117, SIM201, SIM203, SIM300
+
+
     ; for clarity we don't merge 'with Conf():' context trees
-    tests/unit/parsec/*: SIM117, TC001, TC002, TC003
+    tests/unit/parsec/*: B017, B904, C408, C413, SIM113, SIM117, TC001, TC002, TC003
```

Comment:
Rather than ignoring specific codes, can we just ignore the flake8 extensions as a whole.

E.g:

```
tests/*: SIM
```

Rather than:

```
tests/*: SIM201, SIM203, SIM300
```

Reply:
Except Bugbear 904, apparently.